src/babylon.node.ts

```diff
@@ -346,12 +346,12 @@
         public getDirection(localAxis:BABYLON.Vector3): BABYLON.Vector3 {
             var result = BABYLON.Vector3.Zero();
 
-            this.getDirectionToRef(localAxis, result);
+            this.getDirectionToRef(result, localAxis);
             
             return result;
         }
 
-        public getDirectionToRef(localAxis:BABYLON.Vector3, result:BABYLON.Vector3): void {
+        public getDirectionToRef(result:BABYLON.Vector3, localAxis:BABYLON.Vector3): void {

```

Comment:
Optional params have to go on the end and there are several functions in the Bone class that have optional Space and Mesh.

To keep the Bone class consistent I figured it would be a good to do the same with Bone.getDirectionToRef.  I also didn't like how the ref was forced to the middle due to extending Node.
Bone.getDirectionToRef(localAxis, result, mesh);